R/tax_unique.R

```diff
@@ -29,11 +29,16 @@
 #' occurrence dataframe is of this type.
 #' @param resolution \code{character}. The taxonomic resolution at which to
 #' identify unique occurrences, either "species" (the default) or "genus".
+#' @param orig \code{logical}. Should the original dataframe be returned with
```

Comment:
I can't seem to get the new functionality to work with different resolutions, for example:

```
occdf <- tetrapods[1:100, ]
genera_orig <- tax_unique(occdf = occdf, genus = "genus", family = "family",
                     order = "order", class = "class", resolution = "genus", orig = TRUE)
```